source/includes/_lips_booking_events.md

```diff
@@ -0,0 +1,129 @@
+### **BookingEvents**
+<a name="lips_booking_events"></a>
+## Booking Events API
+
+Also referred to as "contact interactions", *events* are activities associated with a contact.  The event is a booking event.
+
+### Resource Overview
+
+| Method | URI Format |
+|---|---|
+| GET `show` |events/:id|
+| PUT `update` | /events/:id |
+| PUT `update` | /events/:id/unarchive |
+| DELETE `destroy` | /events/:id |
```

Comment:
isn't the route [booking_events](https://github.com/GannettDigital/lead-ingestion-publication-service/pull/200/files#diff-959bc9abc46a55332bb64d5155a79323afa75a50ec1a2137ddd22d926f62c6c5R19)?

And I believe we should elaborate that this isn't the event id, but the booking_id.  If you want to interact with the booking event using the LIPS event id, this isn't the API for that.